.circleci/config.yml

```diff
@@ -0,0 +1,24 @@
+# Golang CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-go/ for more details
+version: 2
+jobs:
+  test:
+    docker:
+      - image: circleci/golang:1.13
```

Comment:
Add go1.12?

Reply:
We don't need as we are building a binary. It's not a library

Reply:
Nevertheless useful for contributors. Also, some package manager need a client side compilation.